package.json

```diff
@@ -87,7 +87,8 @@
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "homerunner-client": "^0.0.6",
-    "jasmine": "^3.99.0",
+    "jasmine": "^4.2.1",
```

Comment:
At some point I want to toss out jasmine and replace with jest to broadly be aligned with the rest of matrix-org, but for now any improvements are good.